# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/webgpu/Texture.h
# Commit: c400464776b4
# Full Hash: c400464776b40839926cc31272254e4e93ea8dc8
# Author: Nicolas Silva <nsilva@mozilla.com>
# Date: 2024-04-11 21:36:48
# Description:
#   Bug 1890219 - Null-check the device in TextureView::Cleanup. r=webgpu-reviewers,teoxoy
#   
#   Also Rename Texture::GetParentDevice into Texture::GetDevice.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D206888
# ==============================================================================

diff -r 41fc5b7b0cad -r c400464776b4 dom/webgpu/Texture.h
--- a/dom/webgpu/Texture.h	Thu Apr 11 13:01:25 2024 +0000
+++ b/dom/webgpu/Texture.h	Thu Apr 11 13:19:31 2024 +0000
@@ -35,7 +35,7 @@
 
   Texture(Device* const aParent, RawId aId,
           const dom::GPUTextureDescriptor& aDesc);
-  Device* GetParentDevice() { return mParent; }
+  Device* GetDevice() { return mParent; }
   const RawId mId;
   const dom::GPUTextureFormat mFormat;
   const Maybe<uint8_t> mBytesPerBlock;